src/main/java/Dog.java

```diff
@@ -23,6 +24,7 @@ public void run(){
     public  void stop(){
         System.out.println(name+" stopped");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS106)